Source/SwiftLintFramework/Rules/FileHeaderRule.swift

```diff
@@ -51,9 +51,7 @@ public struct FileHeaderRule: ConfigurationProviderRule, OptInRule {
             }
 
             // skip SwiftLint commands
-            guard !isSwiftLintCommand(token: token, file: file) else {
-                continue
-            }
+            guard !isSwiftLintCommand(token: token, file: file) else { continue }
```

Comment:
This doesn't help with readability.

Reply:
I agree. Any suggestion to decrease the number of lines there to avoid throwing a violation because of the number of lines?

Reply:
Could split up into multiple functions.

Reply:
Reverted.